astropy/stats/__init__.py

```diff
@@ -11,14 +11,14 @@
 
 """
 
-from .funcs import *
-from .biweight import *
-from .sigma_clipping import *
-from .jackknife import *
-from .circstats import *
-from .bayesian_blocks import *
-from .histogram import *
-from .info_theory import *
-from .lombscargle import *
-from .spatial import *
-from .bls import *
+from .funcs import *  # noqa
```

Comment:
fyi I'm planning to do a big cleanup of the inits for 4.0, something similar to what numpy does, but haven't yet got to it yet.